File: codeforces/357.2/A.js
;(function () {
	var n = parseInt(readline())

	for (var i = 1; i <= n; i++) {
		var s = readline().split(' '),
			x = parseInt(s[1]),
			y = parseInt(s[2]);

		if (x >= 2400 && y > x)
			return print("YES");
	}
	print("NO");
})();

File: codeforces/357.2/A.py
n = int(input())
i = 1
while i <= n:
	s, x, y = input().split(' ')
	x = int(x)
	y = int(y)
	if (x >= 2400) and (y > x):
		print('YES')
		exit(0)
	i = i+1
print('NO')